d_free_book/manager.py

```diff
@@ -52,6 +53,11 @@ def create_member(club_id, full_name, code, phone_number=None):
         phone_number=phone_number
     )
 
+
+def check_member_have_phone_number(phone_number):
+    return DFreeMember.objects.filter(phone_number=phone_number)
```

Comment:
add `unique_together = ['club', 'code']` to model
rename func to `def validate_member(data)` then follow style of `validate_oder`